src/api/app/assets/stylesheets/webui2/datatables.scss

```diff
@@ -26,6 +26,18 @@ table.dataTable.table-sm {
 
   td { 
     white-space: normal !important;
-    word-break: break-all;
+
+
+    // hyphens isn't supported on Desktop chrome
```

Comment:
We are using the same code in more places/files. Maybe is a good idea to extend/inherit it. We could do someting like:

``` scss

%hyphens-support {
    // hyphens isn't supported on Desktop chrome
    // https://caniuse.com/#search=hyphens
    @supports(hyphens: auto) {
      hyphens: auto;
    }

    @supports not(hyphens: auto) {
      // break-word is deprecated. But works on Chrome well
      // https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
      word-break: break-word;
    }
}

td {
 @extend %hyphens-support;
}

Reply:
Good suggestion! It looks like it might not be enough, see my [comment](https://github.com/openSUSE/open-build-service/pull/8133#issuecomment-523025213).

However, we definitely need to stop repeating ourselves. I won't merge this PR until we have something fully working.

Reply:
In the end, I'll go with only `break-word`, so this won't be needed.